kakmerge

```diff
@@ -190,6 +196,10 @@ cmd_merge() {
         eval -client base edit! -readonly %{${BASE}}
         eval -client remote edit! -readonly %{${REMOTE}}
 
+        eval -client local 'set-option window autoinfo \"\"'
+        eval -client base 'set-option window autoinfo \"\"'
+        eval -client remote 'set-option window autoinfo \"\"'
```

Comment:
getting rid of (1) on the screenshot

Reply:
If the user has `autoinfo` set, will this hide the information window on subsequent key strokes (e.g. when the user manually moves around the buffer)?

Reply:
I do have autoinfo set myself, my assumption is that as I user I keep my focus in the big window with the actual file, and I don't put my focus in the small readonly windows so I want to keep those readonly windows as clutter-free as possible.

Does it make sense to you?

Reply:
I understand, but I don't think we should be touching user configuration in that case.

Reply:
Remove this hunk.

Reply:
I think we misunderstood each other, let's try again.

I use `autoinfo` and I like it, so I don't just want to globally disable this feature.

Suppose my screen is not as wide, maybe I'm working on a laptop. I use `kakmerge`, and immediately after kakoune opens I see this:

![image](https://user-images.githubusercontent.com/1177900/48917064-327e5000-ee85-11e8-98ef-309b95a7b6d4.png)

The fact that info popup clutters like 20% of the real estate of each of the three bottom windows I consider a bug in `kakmerge`. I didn't press `n`, so I should not see a help message for `n` shortcut. Same if I run `kakmerge-next-conflict`, I'm physically not pressing `n`, so I don't want this message.

As to this specific way of solving this bug, I feel it's reasonable because I expect users to work in the top window, the 3 bottom windows are "helper windows" and it's okay to keep them readonly and without autoinfo.

But if you have different suggestions how to solve this bug without globally disabling `autoinfo`, please say so.

Reply:
Actually when I use `test.sh` (or run the tool in an actual merge, to the best of my recollection), I don't see any of the issues that this PR fixes. I have all four panes open, no autoinfo popup (the option has the default value).

I do see a message that the search was wrapped around the buffer in the relevant panes when  I use `kakmerge-next-conflict` a few times.

Reply:
This is because in my `kakrc` I have this:

```
set-option -add global autoinfo normal
```

But as I said I like this option and don't want to globally disable it, but it doesn't make sense that `kakmerge` generates info messages for me explaining keystrokes that I didn't press.

If this doesn't break your workflow and improves mine, can we merge this and see if there's at least one person who speaks up and tells us why disabling `autoinfo` is bad for them?

Reply:
You can fix this by adding the following to your configuration:

```
hook global WinCreate .+_(LOCAL|BASE|REMOTE)_.+ %{
    set-option window autoinfo ""
}
```

I want the tool to modify the user configuration as little as possible.

Reply:
This works 👍